src/components/AddRequest/AddRequest.scss

```diff
@@ -0,0 +1,22 @@
+@import './src/css/styles';
+
+.add-teacher-report-component {
+  border: 0.06666667rem solid #707070;
```

Comment:
@adrian-goe should we use "0.066rem" instead of these long numbers?

Reply:
Yeah, I'm trying to lay down a rule in linting.